actiontext/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Allow Attachables to override default template when the attachment is missing.
```

Comment:
Could you provide an example in the changelog? :bow:

Reply:
Updated, thanks!